lib/bullhorn/users.ex

```diff
@@ -62,10 +62,11 @@ defmodule Bullhorn.Users do
   end
 
   defp user_notification_method(user, event_type) do
+    credentials = GRPC.Credential.new([])
     request =
       NotificationMethodRequest.new(event_type: event_type, request_id: Bottle.RequestId.write(:rpc), user: user)
 
-    with {:ok, channel} <- GRPC.Stub.connect(account_service_url(), interceptors: [GRPC.Logger.Client]),
+    with {:ok, channel} <- GRPC.Stub.connect(account_service_url(), cred: credentials, interceptors: [GRPC.Logger.Client]),
```

Comment:
Since we aren't using the captured value and we already rely on functions in the call, I think we should move the `GRPC.Credential.new([])` so it stays where it's most relevant. As we change these functions and add code, this piece of important info won't wander away.

It'll also make it easier when copying this to other services so we don't run into this again.

```suggestion
    with {:ok, channel} <- GRPC.Stub.connect(account_service_url(), cred: GRPC.Credential.new([])
, interceptors: [GRPC.Logger.Client]),
```